__tests__/screens/DetailedViewScreen.test.tsx

```diff
@@ -2,8 +2,10 @@ import React from "react";
 import { cleanup, render, fireEvent, RenderAPI } from "@testing-library/react-native";
 import axios, { AxiosResponse } from "axios";
 import { useNearbyPlaces } from "../../src/hooks/useNearbyPlaces";
-import { PlaceWithAccesibilityData } from "../../src/util/placeTypes";
-import { PlaceDetailsResponseData } from "@googlemaps/google-maps-services-js";
+import {
+	PlaceDetailsWithAccesibilityData,
+	PlaceWithAccesibilityData,
+} from "../../src/util/placeTypes";
```

Comment:
btw this should be coming from `peer-types`. I think for now you can keep this bc I'm doing the refactor in #294

Reply:
yeah that's why I left it